sphinx_togglebutton/_static/togglebutton.css

```
@@ -0,0 +1,117 @@
+/**
+ * Admonition-based toggles
+ */
+
+/* Visibility of the target */
+.admonition.toggle .admonition-title ~ * {
+    transition: opacity .3s, height .3s;
+}
+
+/* Overrides for admonition toggles */
+
+/* Toggle buttons inside admonitions so we see the title */
+.toggle.admonition {
+    position: relative;
+}
+
+/* Titles should cut off earlier to avoid overlapping w/ button */
+.toggle.admonition .admonition-title {
+    padding-right: 25%;
+    cursor: pointer;
+}
+
+/* hides all the content of a page until de-toggled */
+.toggle-hidden.admonition .admonition-title ~ * {
+    height: 0;
+    margin: 0;
+    float: left; /* so they overlap when hidden */
+    opacity: 0;
+    visibility: hidden;
+}
+
+/* General button style and position*/
+button.toggle-button {
+    /**
+     * Background and shape. By default there's no background
+     * but users can style as they wish
+     */  
+    background: none;
+    border: none;
+    outline: none;
+
+    /* Positioning just inside the amonition title */
+    margin-right: 0.5em;
+    position: absolute;
+    right: 0em;
+    top: .5em;
+    float: right;
+    width: 1.5em;
+    height: 1.5em;
+    padding: 0px;
+    display: flex;
+}
+
+/* Display the toggle hint on wide screens */
+@media (min-width: 768px) {
+    button.toggle-button.toggle-button-hidden:before {
+        content: attr(data-toggle-hint);  /* This will be filled in by JS */
+        position: absolute;
+        font-size: .8em;
+        left: -6.5em;
+        bottom: .4em;
+    }
+}
+
+/* Icon behavior */
+.tb-icon {
+    transition: transform .2s ease-out;
+    height: 1.5em;
+    width: 1.5em;
+    stroke: currentColor;  /* So that we inherit the color of other text */
+}
+
+.toggle-button-hidden .tb-icon {
+    transform: rotate(90deg);
+}
+
+/**
+ * Details-based toggles.
+ * In this case, we wrap elements with `.toggle` in a details block.
+ */
+
+/* Details blocks */
+details.toggle-details {
+    margin: 1em 0;
+}
+
+details.toggle-details summary {
+    display: flex;
+    align-items: center;
+    width: fit-content;
+    list-style: none;
+    border-radius: .2em;
```

```suggestion
    border-radius: .4em;
```

brings this a bit more in-line with github's button style